### PR TITLE
Topbar redesign

### PR DESCRIPTION
**Pull request by sin6pi7**
- Move NavBar outside of body container
- Redesign top banner
- Update spacing and colors
- Update icons

### CODE REVIEW

**Review by krzkaczor — APPROVED** [review]
I reviewed only visual aspect of this PR 👍

**Review by sz-piotr — CHANGES_REQUESTED** [review]
The site scrolls horizontally on smaller screens
Screenshot 2022-09-15 at 12 18 51

**Comment by sz-piotr** [comment]
Also on mobile the sidebar doesn't align with the banner
<table>
<tr>
<td>
Screenshot 2022-09-15 at 12 26 55
</td>
<td>
Screenshot 2022-09-15 at 12 26 34
</td>
</tr>
</table>

**Comment by sin6pi7** [comment]
@sz-piotr fixed both

File: packages/frontend/src/components/navbar/SidebarMenu.tsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ export function SidebarMenu() {
       <div
         id="sidebar-menu"
         className={classNames(
-          'hidden fixed top-0 left-0 z-50',
+          'hidden fixed top-[32px] left-0 z-50',
```
Comment (sz-piotr) [inline packages/frontend/src/components/navbar/SidebarMenu.tsx:14]: Would be good to have this conditional on config. Also 32px is part of tailwind's scale so you could use top-8

**Review by sz-piotr — CHANGES_REQUESTED** [review]

**Review by sz-piotr — APPROVED** [review]

**Pull request merged**